extension-style/src/main/java/com/mapbox/maps/extension/style/sources/SourceExt.kt

```diff
@@ -17,11 +17,11 @@ import com.mapbox.maps.extension.style.utils.silentUnwrap
 fun StyleManagerInterface.getSource(sourceId: String): Source? {
```

Comment:
Not in this release, but would consider to directly extend `StyleInterface` in the next major release.

Reply:
👍